### PR TITLE
backport css file and filtered table index (#838 #835) 

### PR DESCRIPTION
**Pull request by FredLL-Avaiga**
* #838 help enterprise check the app css file
 * #835 filtered table index

### CODE REVIEW

File: src/taipy/gui/gui.pyi [hunk 1]
```diff
@@ -227,6 +226,8 @@ class Gui:
     def _register_data_accessor(self, data_accessor_class: t.Type[_DataAccessor]) -> None: ...
     def get_flask_app(self) -> Flask: ...
     def _set_frame(self, frame: FrameType): ...
+    __css_file: Incomplete
```
Comment (FabienLelaquais) [inline src/taipy/gui/gui.pyi:229]: ???

**Review by FabienLelaquais — APPROVED** [review]
Excellent

File: src/taipy/gui/gui.pyi (hunk 1 above)
Comment (FredLL-Avaiga) [reply src/taipy/gui/gui.pyi:229]: This is the work of the github action ...

**Review by dinhlongviolin1 — APPROVED** [review]
tx Fred

**Pull request merged**